shipshape.yml

```diff
@@ -172,21 +171,14 @@ checks:
         - docker-compose.yml
       ignore-missing: true
     - name: '[FILE] Yaml lint theme files'
-      severity: high
-      path: web/themes
-      pattern: ".*.yml"
-      exclude-pattern: node_modules
-      ignore-missing: true
-    - name: '[FILE] Yaml lint theme files (no web prefix)'
       severity: high
       path: themes
-      pattern: ".*.yml"
+      pattern: '.*\.yml$'
       exclude-pattern: node_modules
       ignore-missing: true
   phpstan:
     - name: '[FILE] Banned PHP function list'
       severity: high
       configuration: vendor/govcms/scaffold-tooling/phpstan.neon
```

Comment:
With the change to workdir; I think this needs to be relative.

```
      binary: ../vendor/bin/phpstan
      configuration: ../vendor/govcms/scaffold-tooling/phpstan.neon
```

Without this I'm getting;

```
$ ahoy ship-shape             
Breaches were detected!

### [FILE] Banned PHP function list
   -- ./vendor/bin/phpstan: no such file or directory
```